agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -516,7 +516,8 @@ int main(int argc, char *argv[])
     std::unordered_map<int, std::string> interfaces_map;
     for (size_t i = 0; i < num_of_interfaces; i++) {
         if (beerocks::net::network_utils::linux_iface_exists(interfaces[i].ifname)) {
-            interfaces_map[interfaces[i].radio_num] = interfaces[i].ifname;
+            LOG(DEBUG) << "radio" << i << ".hostap_iface=" << std::string(interfaces[i].ifname);
```

Comment:
Why do you create an `std::string` just for the logging?
Pretty sure easylogging++ can logs `char[]` just as good.

Reply:
this change was part of debugging an empty print of hostap_iface read from prplmesh config.
the source of the issue was located elsewhere - but forgot to revert this change.

will change it back to `char[]`